app/helpers/current_user_helper.rb

```diff
@@ -1,9 +1,23 @@
 module CurrentUserHelper
```

Comment:
There's some duplication here. Will revisit after beta's gone, I think most of it can go.